include/mrdox/Config.hpp

```diff
@@ -35,7 +35,7 @@ class ThreadPool;
     particular directory from which absolute paths
     are calculated from relative paths.
 */
-class MRDOX_DECL
+class MRDOX_VISIBLE
```

Comment:
What is the rationale for changing these?

Reply:
Oh. I thought they need to be exported to the dll in general.

Reply:
Yes. MRDOX_DECL does that already. MRDOX_VISIBLE. In doubt, MRDOX_DECL is correct 99% of the time.